From 344fda03fcd696984a26b0aab2620775fdf37f2a Mon Sep 17 00:00:00 2001
From: Bjorn Andersson <bjorn.andersson@linaro.org>
Date: Fri, 8 Nov 2019 22:41:26 -0800
Subject: [PATCH 05/68] arm64: dts: qcom: c630: Add wifi node and regulators

Enable the &wifi node and specify its regulators, as well as specify
voltage (based on the MTP) for the supply regulator.

Signed-off-by: Bjorn Andersson <bjorn.andersson@linaro.org>
---
 arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts b/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts
index 03fd4fb9c0d6e..39466cb9ada5c 100644
--- a/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts
+++ b/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts
@@ -34,9 +34,12 @@ pm8998-rpmh-regulators {
 		qcom,pmic-id = "a";
 
 		vdd-l2-l8-l17-supply = <&vreg_s3a_1p35>;
+		vdd-l4-l5-supply = <&vreg_s7a_1p025>;
 		vdd-l7-l12-l14-l15-supply = <&vreg_s5a_2p04>;
 
 		vreg_s2a_1p125: smps2 {
+			regulator-min-microvolt = <1100000>;
+			regulator-max-microvolt = <1100000>;
 		};
 
 		vreg_s3a_1p35: smps3 {
@@ -58,6 +61,8 @@ vreg_s5a_2p04: smps5 {
 		};
 
 		vreg_s7a_1p025: smps7 {
+			regulator-min-microvolt = <900000>;
+			regulator-max-microvolt = <1028000>;
 		};
 
 		vdd_qusb_hs0:
-- 
2.26.2

